app/assets/styles/app.less

```diff
@@ -43,3 +43,7 @@
 
 // Customizations
 @import './customization/espoo/customization';
+
+// Leaflet
+// React-Select
+@import '~leaflet/dist/leaflet.css';
```

Comment:
- [ ] This should be moved higher in the file where the other third party css imports are located.